poetry.lock

```diff
@@ -3096,19 +3090,18 @@ win32 = ["pywin32"]
 
 [[package]]
 name = "setuptools"
-version = "69.5.1"
+version = "70.0.0"
```

Comment:
downgrade?

Reply:
I would downgrade to keep everything in the same version, reason: to avoid surprises, but not a big thing.